### DISCUSSION TITLE
Usa Telegram para enviar tus nuevas publicaciones

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#29] [Announcements]
# Usa Telegram para enviar tus nuevas publicaciones

Como crear un bot de telegram para enviar mis nuevas publicaciones a un canal de telegram gratis

https://www.zkreations.com/2023/12/como-crear-bot-telegram-para-enviar-feeds-de-blogger-wordpress.html

**Comment by danieIabel** [comment]
He actualizado el código javascript para verificar y guardar solo el ID del artículo, ya que al comparar todos los elementos de la entrada, el mas mínimo cambio provoca que se vuelva a enviar lo cual puede resultar molesto. Con esta modificación solo se enviará si el articulo realmente es distinto.

**Comment by danieIabel** [comment]
He agregado una pequeña verificación para convertir la id a un número cuando se trata de wordpress, ya que de lo contrario la comparación nunca sería correcta.

**Comment by danieIabel** [comment]
Nuevamente, he realizado una actualización. Ya no es necesario obtener el ID del canal manualmente. Tan solo debes agregar tu bot a algún canal como administrador y ya con eso, empezará a enviar actualizaciones de tu contenido a esos canales.

![image](https://github.com/zkreations/data/assets/40010778/fdf2d0eb-4a03-4c81-a095-b58d0d6ef805)
